README.md

```diff
@@ -2,6 +2,48 @@
 
 Where things come from in Who's On First.
 
+## Source Example
+
+**See:** The `whosonfirst-sources` [template file](https://github.com/whosonfirst/whosonfirst-sources/blob/master/source_template.json) when adding a new source.
+
+#### Atlanta Department of Planning and Community Development
```

Comment:
I put this source in as an example, but I'm realizing now that it may be more useful to just have the source property descriptions and a template file..